app/assets/javascripts/datarepo/funder.js

```diff
@@ -29,6 +29,7 @@ $(".simple_form").validate({
 			error.insertAfter( element );
 	}
 });
-$(funderClass).rules("add", {datarepoFunder: ".*:.*"});
-
+if ($(funderClass).length > 0) {
```

Comment:
I don't think so, we don't have multiple parts that are being combined, it's just one test. Sufia does this as `if $(class).length {`